File: EvoHtmlToPdf-v8/Samples/WinForms/CS/EvoHtmlToPdfDemo/HTML_to_PDF/Flash_and_Extensions/Convert_HTML_with_Flash_to_PDF_Sample_Code.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Windows.Forms;

namespace EvoHtmlToPdfDemo.HTML_to_PDF.Flash_and_Extensions
{
    public partial class Convert_HTML_with_Flash_to_PDF_Sample_Code : UserControl
    {
        public Convert_HTML_with_Flash_to_PDF_Sample_Code()
        {
            InitializeComponent();
        }

        private void Convert_HTML_with_Flash_to_PDF_Sample_Code_Load(object sender, EventArgs e)
        {
            codeSampleWebBrowser.Visible = true;
            string codeSampleString = System.IO.File.ReadAllText(@"DemoAppFiles\Input\Code_Samples\CSharp\WinForms\HTML_to_PDF\Flash_Extensions\HTML_Flash_to_PDF.html");
            codeSampleWebBrowser.DocumentText = codeSampleString;
        }
    }
}

File: EvoHtmlToPdf-v8/Samples/WinForms/VB/EvoHtmlToPdfDemo/HTML_to_PDF/Flash_and_Extensions/Convert_HTML_with_Flash_to_PDF_Sample_Code.vb
Imports System.ComponentModel
Imports System.Text

Namespace EvoHtmlToPdfDemo.HTML_to_PDF.Flash_and_Extensions
	Partial Public Class Convert_HTML_with_Flash_to_PDF_Sample_Code
		Inherits UserControl
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Convert_HTML_with_Flash_to_PDF_Sample_Code_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			codeSampleWebBrowser.Visible = True
			Dim codeSampleString As String = System.IO.File.ReadAllText("DemoAppFiles\Input\Code_Samples\VbNet\WinForms\HTML_to_PDF\Flash_Extensions\HTML_Flash_to_PDF.html")
			codeSampleWebBrowser.DocumentText = codeSampleString
		End Sub
	End Class
End Namespace
